### ISSUE TITLE
Editing connections headers does not persist

### ISSUE THREAD

**Issue by paulgessinger** [paulgessinger/swift-paperless#130]
It seems editing extra headers for an existing connection does not affect the current session, maybe also not the stored connection.

**Comment by paulgessinger** [comment]
@leepfrog-ger I wasn't able to reproduce this on my end. When you hit the back button after editing the headers, the app should reload the entire connection and reinitialize everything. I see this in my logs, and I see HTTP requests going out with the custom headers.

Can you try again updating the headers in an existing connection, and share logs with me from right after this?

**Comment by leepfrog-ger** [comment]
I just tried to repro it (by setting a different/wrong header) and this immediately took effect (this however was with 1.5.0, I observed the initial issue with the previous version - and the connection I edited was added quite a while back, possibly even with 1.0).

With 1.5.0 I noticed that indeed there was a "reload" (indicated by shortly showing the logo before returning to a connected state). I cannot recall that behavior with my initial tests.

I don't think it is much of an issue, especially because it cannot be reproduced right now .
